backend/src/Entity/ChatRoomEntity.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ChatRoomEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Uid\Uuid;
+
+#[ORM\Entity(repositoryClass: ChatRoomEntityRepository::class)]
+class ChatRoomEntity
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private $id;
+
+    #[ORM\Column(type: 'integer')]
+    private $userId;
+
+    #[ORM\Column(type: 'integer', nullable: true)]
+    private $usedAs;
+
+    #[ORM\Column(type: 'datetime')]
+    private $createdAt;
```

Comment:
I think you can now use Timestample of Gedmo after we configured it.

Reply:
It is used for any user who owns a chat room, so there is no need to specify the type of user, this router is not currently used, it is only available for the time of need